src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -76,7 +76,7 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   ngOnInit() {
-    this.signupStage = SignUpStage.BASIC_INFO;
```

Comment:
@sri-pusuluri 
please revert this change; on init the stage should be BASIC_INFO